test/src/indicators/MaterialTestIndicator.C

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "MaterialTestIndicator.h"
+#include "Assembly.h"
+
+template<>
+InputParameters validParams<MaterialTestIndicator>()
+{
+  InputParameters params = validParams<Indicator>();
+  params += validParams<MaterialPropertyInterface>();
+  params.addParam<MaterialPropertyName>("property", "The name of the material property to use for an indicator.");
+  return params;
+}
+
+
```

Comment:
Most of the framework only has a single space between function definitions. This file has two